### DISCUSSION TITLE
Air Quality card color coding

### DISCUSSION THREAD

**Discussion by parmiig** [breezy-weather/breezy-weather#1317] [Ideas]
### Describe your idea

Adjust the color code to better suit the AQI number. It seems that it doesn't match the standard thresholds. For instance, the current AQI is 34 which is basically the lowest tier of air pollution but the color in the card is yellow. This seems a bit misleading, as it makes you think air quality isn't good, while it actually is marked Excellent/Fair.

### Other details

Screenshot of the card in BreezyWeather, yellow.
![Screenshot_20240919_180742_Breezy Weather](https://github.com/user-attachments/assets/0ddbe6f2-b724-43f6-9a89-04aef17dd013)

Screenshot of the Weather app in IOS, green.
![2024-09-19 18 16 29](https://github.com/user-attachments/assets/15c298b3-289d-42f6-88bb-0938eda7ba72)

Similar behavior in other Weather apps, like Microsoft's own implementation.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
The implementation is correct; we just don't use the same scale (EAQI vs Plume AQI).

Our scale is universal and very simple to understand:
https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md#current-air-quality
